public/contribute.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "portal-alpha",
+  "description": "File Sharing Experiment",
+  "repository": {
+    "url": "https://github.com/mozilla/something-awesome/",
+    "license": "MPL-2.0"
+  },
+    "participate": {
+      "home": "https://github.com/mozilla/something-awesome/blob/master/README.md",
+      "docs": "https://github.com/mozilla/something-awesome/blob/master/README.md"
+    },
+    "bugs": {
+      "list": "https://github.com/mozilla/something-awesome/issues",
+      "report": "https://github.com/mozilla/something-awesome/issues/new"
+    },
+    "urls": {
+      "prod": "https://fileshare.stage.mozaws.net/",
```

Comment:
Now that we have a final name (Send), I assume this can be updated to "send.firefox.com", per #20.